test-interface/src/main/scala/java/lang/ClassLoader.scala

```diff
@@ -1,19 +0,0 @@
-package java.lang
-
-class ClassLoader protected (parent: ClassLoader) {
```

Comment:
It seems like like actually need the ClassLoader type, even though it's never actually used in our code (only passed in arguments). 
It think we should put it back to javalib, but let's remove the stubs from it. We can leave only the constructors like Scala.js does: 
https://github.com/scala-js/scala-js/blob/2da5417f692955dfc9311e276294d6fa4964f05d/javalib/src/main/scala/java/lang/ClassLoader.scala#L13-L19

Reply:
Yeah, it seems like so, I moved it to `test-interface` and see it works but it doesn't. We at least need a name to pass the reachability check.